tutorials/llm/llama-3/pruning-distillation/04_b_distilling_width_pruned_student.ipynb

```diff
@@ -0,0 +1,138 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d5062f23-c604-479b-9a4e-69989598b131",
+   "metadata": {},
+   "source": [
+    "### Step 4: Distill knowledge from teacher into student\n",
+    "Distillation of a model with NeMo Framework is also possible using a python script: [megatron_gpt_distillation.py](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/megatron_gpt_distillation.py). \n",
+    "In this notebook, we will explore distillation with the width-pruned model as the `STUDENT` model.\n",
+    "\n",
+    "For this demonstration, the `TEACHER` would be the finetuned teacher model `megatron_llama_ft.nemo` and the `STUDENT` model would be the pruned 4B model. This training run is capped by `STEPS`, and validation is carried out every `VAL_INTERVAL` steps."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "be7de691-dd1d-4719-9872-98501a22e3c9",
+   "metadata": {},
+   "source": [
+    "#### Step 4.b.: Using width-pruned student\n",
+    "While distilling knowledge from the teacher to width-pruned model, the `STUDENT` model would be  `4b_width_pruned_model.nemo` as produced by the [width-pruning](./03_b_width_pruning.ipynb) notebook. This training run is capped by `STEPS`, and validation is carried out every `VAL_INTERVAL` steps.\n",
+    "\n",
+    "> `NOTE:` In the block of code below, pass the paths to your pre-processed train, test and validation data files as well as path to the teacher and student .nemo models."
```

Comment:
fix punctuation

 `NOTE:` In the block of code below, pass the paths to your pre-processed train, test, and validation data files, as well as path to the teacher and student .nemo models."